applications/mne_x_libs/xDisp/helpers/selectionmanagerwindow.h

```diff
@@ -42,7 +42,7 @@
 // INCLUDES
 //=============================================================================================================
 
-#include "../ui_selectionmanagerwindow.h"
+#include "ui_selectionmanagerwindow.h"
```

Comment:
>> @@ -42,7 +42,7 @@
>>  // INCLUDES
>>  //=============================================================================================================
>>
>> -#include "../ui_selectionmanagerwindow.h"
>> +#include "ui_selectionmanagerwindow.h"
>
> is this correct?

I did not chang this part. Maybe Chiran does that. However, I think it
belongs to compiling issue.

Also, chiran's rthpi plugin might have problems. We met crash when using
his plugin. But this should not be serious. Let him check his code first.






The information in this e-mail is intended only for the person to whom it is
addressed. If you believe this e-mail was sent to you in error and the e-mail
contains patient information, please contact the Partners Compliance HelpLine at
http://www.partners.org/complianceline . If the e-mail was sent to you in error
but does not contain patient information, please contact the sender and properly
dispose of the e-mail.